plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -917,6 +933,18 @@ private Optional<ConnectorTableExecuteHandle> getTableHandleForOptimize(Connecto
                 icebergTable.location()));
     }
 
+    private Optional<ConnectorTableExecuteHandle> getTableHandleForDropExtendedStats(ConnectorSession session, IcebergTableHandle tableHandle, Map<String, Object> executeProperties)
+    {
+        checkProcedureArgument(executeProperties.isEmpty(), "Unexpected properties");
```

Comment:
nit: quote properties in error message?

Reply:
will do. but that's just some check and the proper checks are done in the engine.
in fact, see `getTableHandleForOptimize`. it gets a propery value from the Map, but doesn't check that the map doesn't have extra entries.

so, to keep code simpler, will just drop the check. this will prevent someone in the future to be forced to do solid validation here